Lob-API-public.yml

```diff
@@ -321,6 +321,106 @@ tags:
       | 422           | SPECIAL_CHARACTERS_RESTRICTED      | Cannot use special characters for merge variable names.                                                              |
       | 422           | UNEMBEDDED_FONTS                   | The provided PDF contains non-standard unembedded fonts. See description for details.                                |
 
+  - name: Events
+    description: |
+      When various notable things happen within the Lob architecture, Events will be created. To get these events sent to your server
+      automatically when they occur, you can set up [Webhooks](https://docs.lob.com/#webhooks).
+
+      **<span style="color:#6683a2">Postcards</span>**
```

Comment:
If you could replace these colored and bolded spans with <h3>, that'd keep styling with the recent UI stuff I pushed consistent.